theia-cpp-docker/Dockerfile

```diff
@@ -101,16 +101,23 @@ RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
                        clang-tools-$LLVM \
                        clangd-$LLVM \
                        clang-tidy-$LLVM \
+                       gcc-multilib \
+                       g++-multilib \
                        gdb && \
-    ln -s /usr/bin/clangd-$LLVM /usr/bin/clangd && \
+    ln -s /usr/bin/clang-$LLVM /usr/bin/clang && \
+    ln -s /usr/bin/clang++-$LLVM /usr/bin/clang++ && \
+    ln -s /usr/bin/clang-cl-$LLVM /usr/bin/clang-cl && \
+    ln -s /usr/bin/clang-cpp-$LLVM /usr/bin/clang-cpp && \
     ln -s /usr/bin/clang-tidy-$LLVM /usr/bin/clang-tidy && \
-    rm -rf /var/lib/apt/lists/*
+    ln -s /usr/bin/clangd-$LLVM /usr/bin/clangd
 
 # Install latest stable CMake
-RUN wget "https://cmake.org/files/v3.17/cmake-3.17.0-Linux-x86_64.sh" && \
-    chmod a+x cmake-3.17.0-Linux-x86_64.sh && \
-    ./cmake-3.17.0-Linux-x86_64.sh --prefix=/usr/ --skip-license && \
-    rm cmake-3.17.0-Linux-x86_64.sh
+ARG CMAKE_VERSION=3.17.3
```

Comment:
Move at the top of the file, along with other similar version-related docker ARGs?

Reply:
This is some kind of design decision... I just kept the style with I found in the file, there where all version defines right before the code block which installs them. This way it is easy to comment complete blocks out to personalize the image to ones need...
But if you prefer, I can put all of them at the top of the file, this way it is easier to see what needs to be updated :-)

Reply:
Fair enough!